tools/hooks/requirements_changed.rb

```diff
@@ -0,0 +1,34 @@
+REPO_DIR = File.expand_path('../../../', __FILE__)
+
+REQUIREMENTS = {
+  "requirements.txt" => "pip install -r requirements.txt",
```

Comment:
nah, it's probably fine, maybe put in a comment here about that in case someone sees this later because they created some unrelated "requirements.txt" file and is confused.